pkg/resourcekeeper/gc_rev.go

```diff
@@ -41,28 +41,20 @@ import (
 	"github.com/oam-dev/kubevela/pkg/utils"
 )
 
-// execute garbage collection functions, including:
-// - clean up legacy app revisions
+// GarbageCollectApplicationRevision execute garbage collection functions, including:
+// - clean up app revisions
 // - clean up legacy component revisions
 func (h *gcHandler) GarbageCollectApplicationRevision(ctx context.Context) error {
 	t := time.Now()
 	defer func() {
 		metrics.AppReconcileStageDurationHistogram.WithLabelValues("gc-app-rev").Observe(time.Since(t).Seconds())
 	}()
-	collectFuncs := []garbageCollectFunc{
-		garbageCollectFunc(cleanUpApplicationRevision),
-		garbageCollectFunc(cleanUpWorkflowComponentRevision),
-	}
-	for _, collectFunc := range collectFuncs {
-		if err := collectFunc(ctx, h); err != nil {
-			return err
-		}
+	if err := cleanUpComponentRevision(ctx, h); err != nil {
```

Comment:
Does the order of recycling component revision and application revision matter?

Reply:
No. The order does not matter.